packages/data-point/README.md

```diff
@@ -848,6 +858,16 @@ A ListReducer is an array of reducers where the result of each reducer becomes t
 | `['$a.b', (input) => { ... }]` | Get path `a.b`, pipe value to function reducer |
 | `['$a.b', (input) => { ... }, 'hash:Foo']` | Get path `a.b`, pipe value to function reducer, pipe result to `hash:Foo` |
 
+An empty `ListReducer` will resolve to `undefined`:
```

Comment:
please add: -> **IMPORTANT**: An empty `ListReducer` will resolve to `undefined`:

or something of that sort, so this part stands out, I can imagine it could lead to devs not knowing why they are getting undefined.

Also if you can add a small/simple explanation of why this behaviour that would help new devs

Reply:
changed it to
> **IMPORTANT**: an empty `ListReducer` will resolve to `undefined`. This mirrors the behavior of empty functions.